mlflow/utils/mlflow_tags.py

```diff
@@ -1,18 +1,27 @@
 """
 File containing all of the run tags in the mlflow. namespace.
 """
+
 MLFLOW_DATABRICKS_NOTEBOOK_ID = "mlflow.databricks.notebookID"
 MLFLOW_DATABRICKS_NOTEBOOK_PATH = "mlflow.databricks.notebookPath"
 MLFLOW_DATABRICKS_WEBAPP_URL = "mlflow.databricks.webappURL"
 MLFLOW_DATABRICKS_RUN_URL = "mlflow.databricks.runURL"
 MLFLOW_DATABRICKS_SHELL_JOB_ID = "mlflow.databricks.shellJobID"
 MLFLOW_DATABRICKS_SHELL_JOB_RUN_ID = "mlflow.databricks.shellJobRunID"
+
 MLFLOW_RUN_NAME = "mlflow.runName"
-MLFLOW_GIT_BRANCH_NAME = "mlflow.gitBranchName"
-MLFLOW_GIT_REPO_URL = "mlflow.gitRepoURL"
+MLFLOW_SOURCE_TYPE = "mlflow.source.type"
```

Comment:
Currently, source type is typed -- NOTEBOOK / JOB / PROJECT / LOCAL / UNKNOWN. With this change, we will lose the documentation on what "built-in" source types are, and what expected values look like. It seems like we should probably have a docs page where we define the default tags for MLflow, similar to our REST API docs.

Reply:
As discussed, I've added this to the REST API docs for now. I'm not sure of meaning of some of the Databricks-specific tags, though - can you have a look and provide descriptions?